src/elements/content-sidebar/activity-feed/ActivityCard.scss

```diff
@@ -1,5 +1,5 @@
 @import '../../common/variables';
 
 .bcs-ActivityCard {
-    padding: $sidebarActivityFeedSpacingVertical $sidebarActivityFeedSpacingHorizontal;
+    padding: 10px $sidebarActivityFeedSpacingHorizontal $sidebarActivityFeedSpacingVertical;
```

Comment:
This will affect the experience for the existing preview experience. Is that intended?

Reply:
Good catch, I've moved this and no changes are done to `ActivityCard.scss`.